main/ertp/api/math-helpers.md

```diff
@@ -1,14 +1,31 @@
-# Math Helpers
-All of the differences in how digital asset amount are manipulated can
-be reduced to the behavior of the math on extents. We extract this
-custom logic into mathHelpers. MathHelpers are about extent
-arithmetic, whereas AmountMath is about amounts, which are the extents
-labeled with a brand. AmountMath use mathHelpers to do their extent
-arithmetic, and then brand the results, making a new amount.
+# MathHelpers
+AmountMath is about doing math operations on `amount`s, which are extents labeled with a brand.
+AmountMath uses MathHelpers to do its extent arithmatic operations. It then brands the results,
+creating a new `amount`. 
+
+There are three different types of MathHelpers, each of which implements all the methods shown 
+on this page. You only have to specify the MathHelper type when creating an `issuer`; it then knows
+which type's operations to use on itself. 
+
+There are three types of MathHelpers, each of which implements all of the same 
+set of API methods (i.e. MathHelpers are polymorphic):
+- `nat`: Used with fungible assests.
+- `strSet`: Used with non-fungible assets.
+- `set`: Used with sets of objects, primarily non-fungible assets.
```

Comment:
I think this isn't explaining the difference between `strSet` and `set` correctly. Both are for nonfungible assets and implement set logic (or to be pedantic, logic similar to set logic). The only difference is that `strSet` has elements that are only strings, but in `set` the elements are records with keys and values. Maybe 
```suggestion
- `strSet`: Used with non-fungible assets, operates on an array of string identifiers
- `set`: Used with non-fungible assets, operates on an array of records (objects) with keys and values
```